python/ray/autoscaler/azure/example-gpu-marcozo.yaml

```diff
@@ -0,0 +1,165 @@
+# An unique identifier for the head node and workers of this cluster.
```

Comment:
do you want this file? if so, recommend renaming it

Reply:
removed file